apollo-router/src/plugins/telemetry/mod.rs

```diff
@@ -976,8 +1140,47 @@ fn handle_error<T: Into<opentelemetry::global::Error>>(err: T) {
     }
 }
 
+pub(crate) fn is_span_sampled() -> bool {
+    Span::current().context().span().span_context().is_sampled()
+}
+
 register_plugin!("apollo", "telemetry", Telemetry);
 
+/// This enum is a partial cleanup of the telemetry plugin logic.
+///
+#[derive(Copy, Clone)]
+enum ApolloFtv1Handler {
```

Comment:
This it not just ftv1 now, so let's call it studio handler or something. We can improve things further later

Reply:
Oh I guess the other changes didn't make it into this enum